src/main/java/hudson/plugins/analysis/core/HealthAwareRecorder.java

```diff
@@ -237,6 +247,10 @@ public HealthAwareRecorder(final String healthy, final String unHealthy,
 
     // CHECKSTYLE:ON
 
+    public HealthAwareRecorder(String pluginName) {
```

Comment:
Note that it would be clearer for this to be marked `protected`: the class is `abstract` so this constructor could only be called by subclass constructors.